frame/balances/src/lib.rs

```diff
@@ -338,13 +338,13 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::force_transfer())]
 		pub fn force_transfer(
 			origin: OriginFor<T>,
-			source: <T::Lookup as StaticLookup>::Source,
-			dest: <T::Lookup as StaticLookup>::Source,
+			source: Box<<T::Lookup as StaticLookup>::Source>,
```

Comment:
curious: what is the static size of this? a bit surprised that it is part of the offending ones.

Reply:
it is 40 bytes if I remember correctly